includes/class-wc-tracker.php

```diff
@@ -616,23 +616,24 @@ private static function get_active_shipping_methods() {
-			'download_require_login'                => get_option( 'woocommerce_downloads_require_login' ),
-			'calc_taxes'                            => get_option( 'woocommerce_calc_taxes' ),
-			'coupons_enabled'                       => get_option( 'woocommerce_enable_coupons' ),
-			'guest_checkout'                        => get_option( 'woocommerce_enable_guest_checkout' ),
-			'secure_checkout'                       => get_option( 'woocommerce_force_ssl_checkout' ),
-			'enable_signup_and_login_from_checkout' => get_option( 'woocommerce_enable_signup_and_login_from_checkout' ),
-			'enable_myaccount_registration'         => get_option( 'woocommerce_enable_myaccount_registration' ),
-			'registration_generate_username'        => get_option( 'woocommerce_registration_generate_username' ),
-			'registration_generate_password'        => get_option( 'woocommerce_registration_generate_password' ),
+			'version'                                                    => WC()->version,
+			'currency'                                                   => get_woocommerce_currency(),
+			'base_location'                                              => WC()->countries->get_base_country(),
+			'selling_locations'                                          => WC()->countries->get_allowed_countries(),
+			'api_enabled'                                                => get_option( 'woocommerce_api_enabled' ),
+			'weight_unit'                                                => get_option( 'woocommerce_weight_unit' ),
+			'dimension_unit'                                             => get_option( 'woocommerce_dimension_unit' ),
+			'download_method'                                            => get_option( 'woocommerce_file_download_method' ),
+			'download_require_login'                                     => get_option( 'woocommerce_downloads_require_login' ),
+			'calc_taxes'                                                 => get_option( 'woocommerce_calc_taxes' ),
+			'coupons_enabled'                                            => get_option( 'woocommerce_enable_coupons' ),
+			'guest_checkout'                                             => get_option( 'woocommerce_enable_guest_checkout' ),
+			'secure_checkout'                                            => get_option( 'woocommerce_force_ssl_checkout' ),
+			'enable_signup_and_login_from_checkout'                      => get_option( 'woocommerce_enable_signup_and_login_from_checkout' ),
+			'enable_myaccount_registration'                              => get_option( 'woocommerce_enable_myaccount_registration' ),
+			'registration_generate_username'                             => get_option( 'woocommerce_registration_generate_username' ),
+			'registration_generate_password'                             => get_option( 'woocommerce_registration_generate_password' ),
+			'woocommerce_registration_generate_password_allow_overwrite' => get_option( 'woocommerce_registration_generate_password_allow_overwrite' ),
```

Comment:
Naming seems inconsistent with other option names above which do not contain "woocommerce".